packages/react-native-client/android/build.gradle

```diff
@@ -77,7 +78,12 @@ repositories {
 dependencies {
   implementation project(':expo-modules-core')
   implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${getKotlinVersion()}"
-  implementation "com.github.fishjam-cloud:android-client-sdk:0.4.0"
+  if (findProject(':android-client')) {
+    implementation project(':android-client')
+  } else {
+//   todo when released, changed repo name etc then update
+    implementation 'com.github.fishjam-cloud:android-client-sdk:0.4.0'
```

Comment:
so do we need to update this manually on every release?

Reply:
it probably can be automated, but firstly we should discuss how we should think of process for deploying all packages (rn, android, ios)

Reply:
I hope we will be able to automate releasing all three SDKs once iOS one will be merged to this repo too...